docs/source/user/limitations.rst

```diff
@@ -1,3 +1,5 @@
+.. _limitations:
```

Comment:
Hmmm that's weird, I thought I had automatic heading links working, did that break?

Reply:
I tried adding it once and autoref complained about duplicate section names all throughout the documentation. Interested to hear how you added it. But no, I can't remove it on the current configuration in my branch.